vortex-api/src/main/java/mil/army/usace/hec/vortex/io/AscDataReader.java

```diff
@@ -237,6 +235,9 @@ public List<VortexData> getDtos() {
         dataset.delete();
         band.delete();
 
+        VortexVariable vortexVariable = VortexVariable.fromName(shortName);
+        // FIXME: What should the data type be?
```

Comment:
What should the `VortexDataType` be, when reading from an ASCII data source?

Reply:
This should be more or less the same as `NetcdfDataReader`. The reader itself is agnostic to the memory representation in `VortexGrid`. See my other comment about calling infer data type in `VortexGrid::getDataType`.